include/fluent-bit/flb_log_event_decoder.h

```diff
@@ -77,7 +80,7 @@ int flb_log_event_decoder_decode_timestamp(msgpack_object *input,
 int flb_event_decoder_decode_object(struct flb_log_event_decoder *context,
                                     struct flb_log_event *event,
                                     msgpack_object *input);
-
+int flb_log_event_decoder_get_decode_result(struct flb_log_event_decoder *context);
```

Comment:
Rename this to `flb_log_event_decoder_get_last_result`

Reply:
I renamed it.